src/components/CreateSpotAndPersona/CreateCard/EntityCard.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { ChangeEvent, FC, useState } from 'react';
```

Comment:
Why EntityCard is not typed as Functional Component anymore? (FC)

Reply:
Because when i have FC i receive error. Fc is unnecessary in this case